script.js

```diff
@@ -0,0 +1,364 @@
+initializeCalculator();
+function initializeCalculator() {
+  let first = {value: "", unsaved: true, saved: false, integer: true};
```

Comment:
`first` or `second` are not very descriptive names and it is not easy to figure out what they refer to just by looking at the variable names. I'd call this variable `firstDigitInEquation` or `secondDigitInEquation` to be more precise.